plugins/DuskVerb/tools/tuner/apply_best_and_evaluate.py

```
@@ -0,0 +1,181 @@
+    centers = centers[(centers > 25) & (centers < sr * 0.4)]
+    out = []
+    for fc in centers:
+        lo, hi = fc / (2 ** (1/6)), fc * (2 ** (1/6))
+        msk = (f >= lo) & (f < hi)
+        e = float(S[msk].sum())
+        out.append((float(fc), 10 * np.log10(e + 1e-30)))
+    return out
+
+
+def report(preset, params, vst3, anchor):
+    dv_dir = Path("/tmp/eval_best")
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/eval_best"

(S108)